.github/workflows/build_binary.yml

```diff
@@ -0,0 +1,116 @@
+name: Binary Release Build
+
+on:
+  push:
+    branches:
+      - release/**
+
+jobs:
+  linux:
```

Comment:
You could use a [matrix](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) here, but tbh what you have is probably more clear despite some duplication

Reply:
Good point. I had a matrix at first, but ended up choosing this for the cleaner config. As soon as `craft` supports GitHub artifacts, the biggest chunk of duplication (zeus uploads) will be gone and this will be neat and clean. Would propose to revisit once we can move off Zeus.